src/cnv/tests/system/test_run_cnvpytor.py

```diff
@@ -0,0 +1,45 @@
+import filecmp
+import warnings
+from os.path import join as pjoin
+from pathlib import Path
+
+import pytest
+from ugbio_cnv import run_cnvpytor
+
+warnings.filterwarnings("ignore")
+
+
+@pytest.fixture
+def resources_dir():
+    return Path(__file__).parent.parent / "resources"
+
+
+class TestRunCNVpytor:
+    def test_run_cnvpytor(self, tmp_path, resources_dir):
+        input_cram_file = pjoin(resources_dir, "HG002.chr19.s0.01.bam")
```

Comment:
```suggestion
        input_cram_file = resources_dir / "HG002.chr19.s0.01.bam"
```

Since resources_dir is a Path object, you can use a simpler syntax

Reply:
yes this does not work for me locally. tried it for other scripts before. 
I get the following error message when trying running a test: 
>       if not arg_string[0] in self.prefix_chars:
E       TypeError: 'PosixPath' object is not subscriptable

keeping it as is.

Reply:
Next time you can convert it to string before calling it in the test:
                "run_cnvpytor",
                "--input_bam_cram_file",
                str(input_cram_file),
                
In general, I prefer working with **Path** objects, this is the Python standard we are aiming for. It is ok for now, but try it next time.